<launch>
    <arg name="auto_trigger" default="true"/> <!-- whether to automatic startup the pipeline at the first data received -->
    <param name="world_frame_id" value="world"/>
    <param name="dvs_bootstrap_frame_id" value="camera0" /> <!-- to substitute svo, change this with the frame published by your pipeline -->
    <param name="dvs_frame_id" value="dvs0" />
    <param name="num_ev_per_map" value="300000" />
    <!-- Mapping module -->
    <node pkg="mapper_emvs_stereo" type="run_emvs" name="run_emvs" output="screen" args="--flagfile $(find mapper_emvs_stereo)/cfg/tracker/alg1/rpg_reader.conf">
	<remap from="pointcloud_local" to="/dvs_mapping/pointcloud" /> 
	<remap from="pointcloud_global" to="/dvs_mapping/pointcloud_global" />
	<remap from="Inverse_Depth_Map" to="/dvs_mapping/depthmap" />  
    </node>
</launch>
